zee_slider.js

```diff
@@ -168,20 +171,45 @@
     }
   }
 
+  function initializeBulletsLinks() {
+    var _ = this,
```

Comment:
thats can be avoided by number of techniques, 
for starter you can use `arrow functions`  you won't have to do this funny stuff of assigning this. 
especially use it on the foreach callback if you want to access the outer function scope easily.